extensions/import/src/wizard/pages/prosePreviewPage.ts

```diff
@@ -70,14 +62,22 @@ export class ProsePreviewPage extends ImportPage {
 			columns: undefined,
 			forceFitColumns: azdata.ColumnSizingMode.DataFit
 		}).component();
-		this.refresh = this.view.modelBuilder.button().withProps({
-			label: constants.refreshText,
-			isFile: false,
-			secondary: true
-		}).component();
 
-		this.refresh.onDidClick(async () => {
-			await this.onPageEnter();
+		this.instance.createDerivedColumnButton.onClick(async (e) => {
+			const derivedColumnDialog = new DerivedColumnDialog(this.model, this.provider);
+			const response = await derivedColumnDialog.createDerivedColumn();
+			if (response) {
+				this.model.proseColumns.push({
+					columnName: response.derivedColumnName,
+					dataType: 'nvarchar(MAX)',
```

Comment:
The derived column is always going to be nvarchar(MAX)? Not datetime or anything else?